### PR TITLE
Refractor: Javascript to ES-6 syntax

### PR DESCRIPTION
**Pull request by abiola-arobieke**
In this milestone:

Index.js file was split into different modules.
books.js  and spa.js file was created.
luxon was installed via npm, and imported into index.js file to show the current date and time.
Javascript functions were refactored to ES-6 syntax.

### CODE REVIEW

File: book.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    if (storedBooks) {
+      this.books = storedBooks.map((book) => ({
+        title: book.title,
+        author: book.author,
+      }));
+    }
+  }
+
+  add(event) {
+    event.preventDefault();
```
Comment (mikethreels) [inline book.js:29]: - [x] According to the requirements you should create a directory called modules and put each separate module you created in this directory.

File: book.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    this.books.push(book);
+    this.titleInputElement.value = '';
+    this.authorInput.value = '';
+    this.updateBookList();
+  }
+
+  removeBook(title) {
+    this.books = this.books.filter((book) => book.title !== title);
+    this.updateBookList();
+  }
```
Comment (mikethreels) [inline book.js:42]: - [x] When removing a book which was duplicated in the list the remove button removes both instances of the book, please make sure that the remove button only removes the book which is selected.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Abiola, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/book.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      listItem.appendChild(removeButton);
+      listItem.appendChild(breakline);
+      this.bookListElement.appendChild(listItem);
+    });
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
+}
+
+// const allRemoveBtn = document.querySelectorAll('.remove-button');
+// console.log(allRemoveBtn);
```
Comment (tufoinnkuo10) [inline modules/book.js:66]: - [optional] nice job. For clean code practices, always remove commented code

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @abiola-arobieke 

Your project is complete! Great job for making the changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- modules well used
- luxon library  well applied
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**